src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
I see. Sadly, PSR-3 does not support that.

We have personally been running this in production for the last 2 years or so. The actual overhead is in writing to the log file though, not the string concatenation.

Is this a blocking issue for you?